core/iwasm/libraries/lib-wasi-threads/tid_allocator.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include "tid_allocator.h"
+#include "wasm_export.h"
+#include "bh_log.h"
+
+#define MIN(a, b) (((a) < (b)) ? (a) : (b))
+
+struct tidAllocator {
```

Comment:
I think struct names should follow `PascalCase` (i.e. `TidAllocator`).

Reply:
I was using `TidAllocator` for the opaque struct in the header (i.e. `typedef struct tidAllocator TidAllocator;`) but actually I can use the same name for both